development/docker-compose.yml

```diff
@@ -38,7 +36,6 @@ services:
     healthcheck:
       interval: 5s
       timeout: 5s
-      start_period: 5s
```

Comment:
Why does this changeset keep appearing in your PRs? Is there something we need to address or fix here?

Reply:
Sorry, left that in again. Whenever i run a invoke command, they dont work saying `target` and `start_period` are unsupported paramenters .

Reply:
Probably you're running an old version of `docker-compose` that doesn't support them - consider updating it?